modelmapper/stats.py

```diff
@@ -409,7 +409,7 @@ def matchers_from_settings(settings=None):
     """
     Creates a default set of matchers from the settings object
     """
-    datetime_formats = settings.datetime_formats if settings else None
+    datetime_formats = settings.datetime_formats if settings else {"%m/%d/%y", "%m/%d/%Y", "%Y%m%d", "%Y-%m-%d"}
```

Comment:
we should either keep this in the `DateTimeMatcher` or move the null/boolean defaults here as well for consistency

Reply:
@JackWink Good point. I'm in favor of moving the defaults into higher level functions. When I was debugging some code, it took me a while to realize the defaults were beings set deep in the stack.